src/fvm_multicell.hpp

```diff
@@ -85,8 +85,56 @@ class fvm_multicell {
         return (this->*h.first)[h.second];
     }
 
-    /// integrate all cell state forward in time
-    void advance(double dt);
+    /// Initialize state prior to a sequence of integration steps.
+    void start_integration(value_type tfinal, value_type dt_max) {
```

Comment:
The name `start_integration` gives the impression that integration will be performed when called.